docs/Client.asciidoc

```diff
@@ -123,11 +123,11 @@ https://en.wikipedia.org/wiki/JSON[JSON]. Both will be referenced extensively.
 The `--method` option (or `-X` for short) allows you to change the HTTP request
 method from `GET` to something else. In the openQA API you will most commonly
 encounter `POST`, `PUT` and `DELETE`. For example to start testing a new ISO
-image you would use `POST`.
+image you would use `POST`:
 
 [source,sh]
 ----
-openqa-cli api --host http://openqa.example.com -X POST isos \
+openqa-cli api --host http://openqa.example.com -X POST isos async=0 \
```

Comment:
IMHO it's not the job of the client documentation to explain what the server should do or does at default. I would simply not state the async parameter. The users guide already explains (with your changes) where async should be applicable

Reply:
So far the example already states all interesting parameters explicitly and it seems consequent to have `async=0` as well. I could of course also just state one parameter, e.g. `ISO=foo.iso …`. We don't really need a full example here.

Reply:
well effectively all other parameters specified in this example are actually needed, unlike `async=0` so I would favor that you simply leave this line as-is